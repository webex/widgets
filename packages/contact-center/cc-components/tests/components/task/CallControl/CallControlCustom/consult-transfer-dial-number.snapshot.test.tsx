import React from 'react';
import {render} from '@testing-library/react';
import DialNumberUI from '../../../../../src/components/task/CallControl/CallControlCustom/consult-transfer-dial-number';

describe('DialNumberUI snapshot', () => {
  const loggerMock = {
    log: jest.fn(),
    info: jest.fn(),
    warn: jest.fn(),
    trace: jest.fn(),
    error: jest.fn(),
    debug: jest.fn(),
  };
  const onButtonPressMock = jest.fn();
  const defaultProps = {
    title: 'Dial Number',
    buttonIcon: 'call',
    onButtonPress: onButtonPressMock,
    logger: loggerMock,
  };

  it('matches snapshot', () => {
    const {container} = render(<DialNumberUI {...defaultProps} />);
    expect(container).toMatchSnapshot();
  });
});
