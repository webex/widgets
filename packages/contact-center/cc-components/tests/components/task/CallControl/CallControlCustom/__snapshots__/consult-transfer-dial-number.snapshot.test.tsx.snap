// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DialNumberUI snapshot matches snapshot 1`] = `
<div>
  <div
    class="consult-transfer-dial-number-container"
  >
    <mdc-input
      class="consult-transfer-dial-number-input"
      data-testid="consult-transfer-dial-number-input"
    />
    <button
      class="md-button-circle-wrapper consult-transfer-dial-number-btn md-button-simple-wrapper"
      data-color="join"
      data-disabled="false"
      data-ghost="false"
      data-inverted="false"
      data-multiple-children="false"
      data-outline="false"
      data-shallow-disabled="false"
      data-size="28"
      data-testid="dial-number-btn"
      type="button"
    >
      <mdc-icon />
    </button>
  </div>
</div>
`;
